docs/get_started.rst

```diff
@@ -8,6 +8,17 @@ Overview
 .. automodule:: django_select2
     :members:
 
+Assumptions
+-----------
+
+* You have a Django project, possibly with models linked by foreign key fields,
```

Comment:
That is the main scenario, but not all of them. Maybe it makes sense to coin it, so it is clear that the library does support other scenarios.

Reply:
I"m not sure I understand your comment.  what about you edit the assumptions as to make them more correct, after you accept the PR?